tools/harness-automation/autothreadharness/settings_sample.py

```diff
@@ -35,6 +35,10 @@
 """(str, str): The first element is serial port of the DUT, and the second is
 the device type. This must be set if AUTO_DUT=False."""
 
+DUT2_DEVICE = ('COM18', 'OpenThread')
+"""(str, str): The first element is serial port of the DUT, and the second is
+the device type. DUT in RF-box for RF-shield needed cases and this must be set if AUTO_DUT=False."""
```

Comment:
why the condition 'if AUTO_DUT=False' here?
I think our existing Automation tool only support AUTO_DUT method regardless of whether or not DUT is shielded. right?

Reply:
Deleted